### ISSUE TITLE
Too many useless "WARNINGS" while docker compose up

### ISSUE THREAD

**Issue by sykp241095** [pingcap/autoflow#261]
```
backend-1     | /usr/local/lib/python3.11/site-packages/pydantic/_internal/_config.py:341: UserWarning: Valid config keys have changed in V2:
backend-1     | * 'orm_mode' has been renamed to 'from_attributes'
```

**Comment by AntiKnot** [comment]
hi @sykp241095 ,


The module relies on pydantic to make v1 v2 compatible. However, **fastapi_users_db_sqlmodel** is not compatible.

This project repo has not been updated for a year. Find the relevant commit in pr.
https://github.com/fastapi-users/fastapi-users-db-sqlmodel/pull/19/files
```python
    if PYDANTIC_V2:  # pragma: no cover
        model_config = ConfigDict(from_attributes=True)  # type: ignore
    else:  # pragma: no cover

        class Config:
            orm_mode = True
```

This issue was resolved after trying to use this PR. 
I also found the same changes in other modules that depend on pydantic to deal with the problem of using pydantic V2.

----
Some complaints
Upgrade to fastapi 2.* and pydantic 2.*. Compatibility issues are common in the dependency ecosystem.
